docs/developers/content-creation/creating-a-weapon.md

```diff
@@ -1,3 +1,169 @@
+SWEP.Primary.Ammo          = "smg1" -- Which ttt ammo entity is picked up by this swep
+SWEP.Primary.ClipSize      = 100 -- Maximum clip size
+SWEP.Primary.DefaultClip   = 100 -- Starting clip size
+SWEP.Primary.Sound         = Sound("Weapon_AK47.Single") -- Either a sound file from the default list or a custom one
+SWEP.Primary.NumShots      = 1 -- Number of shots fired at one time (make into shotgun if increase conde val)
+SWEP.Primary.Recoil        = 1.5 -- Recoil value (bigger = bigger)
+```
+
+!!! note
+    All of these attributes can also be applied to SWEP.Secondary which is alt click (if you disable ironsights).
+
+```lua
+SWEP.HoldType              = "ar2"
+SWEP.ViewModel             = "models/weapons/c_irifle.mdl" -- What the swep looks like to the user (Custom models can be used too)
+SWEP.WorldModel            = "models/weapons/w_irifle.mdl" -- What the swep looks like to everyone else
+SWEP.UseHands              = true -- Whether the swep will force the user to see the viewmodel's hands
+SWEP.AutoSpawnable         = false -- Whether the swep will spawn upon map gen (override as false if EQUIP1/2)
+SWEP.AmmoEnt               = "item_ammo_smg1_ttt" -- Which entity you pick up to fill your clips
+SWEP.InLoadoutFor          = nil -- Which roles gain this weapon upon round start (table data structure)
+SWEP.LimitedStock          = true -- If true only one can be bought per round
+SWEP.AllowDrop             = true -- Is the player able to drop the swep
+SWEP.IsSilent              = false -- If true, the killed player will not scream upon death
+SWEP.NoSights              = false -- If true, the swep has no ironsights capability (no Secondary fire if ironsights?)
+SWEP.HeadshotMultiplier    = 50 -- Multiply the headshot damage by this much
+SWEP.DeploySpeed           = 0.2 -- Lower the slower
+SWEP.IronSightsPos         = Vector(-2, -5, 0) -- Moves the Viewmodel by this vector when using ironsights
+SWEP.IronSightsAng         = Vector(-2, 0, 0) -- Rotates the Viewmodel by this angle when using ironsights
+```
+
+These are most of the other miscellaneous values which can also be applied to the SWEP (That I have found, if any others are found to be applicable to TTT2 please notify us). Note that if any of these values are not explicitly stated in your lua file they will inherit the weapon_tttbase ones, or cause errors.
+
+```lua
+SWEP.Kind                 = WEAPON_EQUIP1 -- Kind specifies what weapon_ttt category it falls into
+--[[
+   (        0      1      2       3      4     5      6       7       8   ) SWEP.Slot
+   (WEAPON_ MELEE, HEAVY, PISTOL, HEAVY, NADE, CARRY, EQUIP1, EQUIP2, ROLE) SWEP.Kind
+--]]
+```
+
++1 to the SWEP.Slot get the eqip slot key
+
+```lua
+SWEP.CanBuy                = {ROLE_TRAITOR} -- Which roles can purchase this swep (table)
+```
+
+This data is stored in a table, its the roles which can purchase it in the buy menu. Convention is that the keyword for each role is ROLE_\<ROLE NAME>.
```

Comment:
This line looks like this:
![image](https://user-images.githubusercontent.com/16392835/86133183-44f04f00-bae8-11ea-8369-9169a4050fdf.png)
with mkdocs.

The following renders correctly for me
```suggestion
This data is stored in a table, its the roles which can purchase it in the buy menu. Convention is that the keyword for each role is ROLE_<ROLE NAME\>.
```

Reply:
thanks, had problems with how mkdocs interpreted the \ escape char :/